.github/workflows/stack.yml

```diff
@@ -12,10 +12,6 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - resolver: lts-14  # GHC 8.6
-            yaml: stack.yaml
```

Comment:
This was here to make sure we test compatibility with the latest hackage release of xmonad and X11 on both oldest and newest supported GHC. So only one of these should be dropped, and the stack.yaml one should be moved to a newer resolver.

Reply:
Ah, so that was what that was :)

Reply:
I'm on it.

Reply:
I should've added a comment really. It's a small miracle I still remember what the point was. Perhaps I don't and what I said isn't even true 😂

Reply:
Looks correct to me from inspecting both the `stack.yaml` files. (I'm reminded of the stuff I added to `cabal`'s CI to do much the same thing, but it's a little more involved since of course they don't build with `stack`.)